lib/chef/provider/package/aix.rb

```diff
@@ -88,19 +88,24 @@ def load_current_resource
 
         def candidate_version
           return @candidate_version if @candidate_version
-          ret = shell_out_with_timeout("installp -L -d #{@new_resource.source}")
-          ret.stdout.each_line do |line|
-            case line
-            when /\w:#{Regexp.escape(@new_resource.package_name)}:(.*)/
-              fields = line.split(":")
-              @candidate_version = fields[2]
-              @new_resource.version(fields[2])
-              Chef::Log.debug("#{@new_resource} setting install candidate version to #{@candidate_version}")
+          if @new_resource.source
+            @package_source_found = ::File.exists?(@new_resource.source)
+            if @package_source_found
```

Comment:
since this is getting copypasta'd around you probably want to consolidate this code now:

```
def package_source_found?
  @package_source_found ||= new_resource.source && ::File.exist?(new_resource.source)
end
```

then you can just `if package_source_found?` here.